volunteers/models.py

```diff
@@ -66,9 +66,12 @@ def __unicode__(self):
     def get_current(cls):
         retval = False
         today = datetime.date.today()
-        current = cls.objects.filter(visible_from__lte=today, visible_until__gte=today)
-        if current:
-            retval = current[0]
+        try:
+            current = cls.objects.filter(visible_from__lte=today, visible_until__gte=today)
+            if current:
+                retval = current[0]
+        except:
```

Comment:
Is this here to catch a specific exception? If so, could that be used instead of the naked `except`